task_1/anna.zinovatnaya/src/main/java/task_1/Commit.java

```diff
@@ -0,0 +1,80 @@
+package main.java;
+
+import org.apache.commons.codec.digest.DigestUtils;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Map;
+
+public class Commit implements Element {
+
+  private String hash;
+  private String parent;
+  private String tree;
+  private String message;
+  private Date date;
```

Comment:
New Java Data API has been introduced in Java 8. It is easier to use and all the classes are immutable compared to **Date**. It is not illegal to use the old **Date** API. But I encourage to have a look at a new one (http://tutorials.jenkov.com/java-date-time/localdate.html) and change your **Date** field to **LocalDate**.